spec/lib/jobs/install-os-job-spec.js

```diff
@@ -379,5 +379,28 @@ describe('Install OS Job', function () {
                    .to.throw('The uid should between 500 and 65535 (>=500, <=65535)');
         });
         
+        it('should throw error when mountPoint is not a string', function() {
+            job.options.installPartitions = [{ mountPoint:{}, size:'500', fsType:'ext3' }];
+            return expect(job._validateOptions.bind(job,{}))
+                   .to.throw('mountPoint must be a string');
+        });
+
+        it('should throw error when size is not a string', function() {
+            job.options.installPartitions = [{ mountPoint:'/boot', size:{}, fsType:'ext3' }];
```

Comment:
@yyscamper  IMHO. I don't think so. this way is not increasing either line coverage or branch coverage. actually, the input such as  null, NAN, etc. is not testing our codes. it's testing assert-plus's 'assert.string()'. although it looks like it tests all kinds of cases. it's missing focus test for our codes. and it's not unit test's intention.